src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1778,16 +1787,42 @@ bool sockinfo_udp::tx_check_if_would_not_block()
     #pragma BullseyeCoverage on
 #endif
 
-bool sockinfo_udp::rx_input_cb(mem_buf_desc_t* p_desc, void* pv_fd_ready_array)
+/**
+ * sockinfo_udp::inspect_uc_packet inspects the input packet for basic rules,
+ * common for all cases. Its applicable for UC case as well.
+ */
+
+inline bool sockinfo_udp::inspect_uc_packet(mem_buf_desc_t* p_desc)
```

Comment:
Re. the port protection check:

1. It relies on the OS providing a different port number when for reopned sockets. It does not solve the problem where the client explicitly requests the same port twice. These points need to be well documented here.
2. Please open a new RM ticket for improving the port protection mechanism (draining all packets belonging to the old socket, possible prohibiting of flowtag usage if a socket is re-opned with the same fd # etc...).